modules/ROOT/pages/troubleshooting-visualization.adoc

```diff
@@ -0,0 +1,87 @@
+= Troubleshooting Visualization
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+3
+
+== About the Troubleshooting Visualization
+
+The Troubleshooting Visualization shows you the current and past health of your Mule apps. It allows you to quickly visualize where your network is slow, receiving heavy traffic or failing. From this visualization, you can drill down directly into Anypoint Monitoring dashboards or logs to continue to troubleshoot the performance of your systems.
+
+== Choosing your Metric
+
+Use the options in the *Metric* card to select the metric that you want to show for all the nodes in your application network. For applicable metrics, the metric value will also appear on the connection, or _edge_.
+
+In the *Metrics* section, you can select one of the following metrics:
+
+* *Avg response time*
+* *Avg throughput*
+* *Failure rate*
+* *Avg CPU utilization*
+* *Avg memory utilization*
+
+For the selected time range, _failure rate_ is equal to the total number of failures divided by the total number of requests.
+
+For average response time, average throughput, and failure rate, you can select *Inbound* or *Outbound* metrics.
+If you select inbound, the value that appears on the application is the weighted average of that particular metric for all inbound connections to that node.
+If you select outbound, the value is the weighted average of that metric for all outbound connections.
+
+In the *Time Range* section, you can select the time window used to calculate the average values of the selected metric.
```

Comment:
I think this might be out of logical order.
We need to set the time range first, right, and then select the metric to show.
And should the text following the list of metrics instead follow each metric list item?
For example, we describe failure rate before the two metrics shown above it.
And we mention a selected time range before describing the time range section.

Reply:
With regards to comment 1, this order follows the order in the UI:
1) Metric choice
2) Inbound or Outbound
3) Time range

Reply:
Oh! OK. Sara, consider saying something, when you first mention the time range, about it being selected later in the process.